tardis/plasma/tests/test_nlte_excitation.py

```diff
@@ -116,3 +117,44 @@ def test_prepare_bound_bound_rate_matrix(
         np.array(actual_rate_matrix),
         rtol=1e-6,
     )
+
+
+@pytest.mark.parametrize(
+    [
+        "coll_exc_coeff_values",
+        "coll_deexc_coeff_values",
+        "desired_coeff_matrix",
+    ],
+    [
+        (
+            [1, -2, 3],
```

Comment:
For testing it doesn't matter but it is a bit odd to have a negative collisional rate. I would at least add an inline comment to state that this is not physical.